docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -42,87 +42,194 @@ Antenatal care attendance module
 
-2.4: Module Outputs
------------------------
+ANC attendance inputs to the :ref:`hemoglobin module <2024_vivarium_mncnh_portfolio_hemoglobin_module>`
+are dichotomous for each pregnancy, so we need to observe the following: 
 
-.. list-table:: Module outputs
+.. list-table:: Module outputs for hemoglobin component
   :header-rows: 1
 
   * - Output
     - Value
     - Note
-  * - A. ANC1?
+  * - Attends ANC in first trimester?
     - *True* / *False*
-    - 
+    - For groups B or D 
+  * - Attends ANC in later pregnancy?
+    - *True* / *False* 
+    - For groups A or C
+
+ANC attendance inputs to the :ref:`facility choice module <2024_vivarium_mncnh_portfolio_facility_choice_module>`
+are polychotomous for each pregnancy with the following 4 categories (ordered from worst to best):
+
+1. No ANC
+2. ANC in later pregnancy only
+3. ANC in 1st trimester only
+4. ANC in 1st trimester and later pregnancy
+
 
 3.0 Assumptions and limitations
 ++++++++++++++++++++++++++++++++
 
-* We assume that partial term pregnancies have the same probability of ANC attendance as full term pregnancies. Notably the ANC1 covariate from DHS is measured among live and stillbirths only. Notably, first trimester ANC attendance tends to be lower than overall pregnancy ANC attendance [Nigeria-DHS]_. Given that partial term pregnancies are by definition <24 weeks in duration, it is possible that they have lower attendance rates than full term pregnancies *prior to the premature end of their pregnancies.* 
+* We assume that partial term pregnancies have the same probability of ANC coverage as full term pregnancies and assume that these visits occur exclusively 
+  during the first trimester given that these pregnancies do not progress late into pregnancy. Note that in effect this makes first trimester ANC attendance 
+  rate higher among partial term pregnancies than full term pregnancies, which is not necessarily logical. We accept this as a limitation given that the 
+  interventions partial term pregnancies can receive at a first trimester visit (MMS, AI ultrasound) primarily affect outcomes that do not apply to partial 
+  term pregnancies (MMS->stillbirth, MMS->LBWSG, MMS->hemoglobin->sepsis/obstructed labor, and ultrasound/facility delivery). However, it is possible that 
+  we will overestimate the impact of MMS->hemoglobin on anemia YLDs and depressive disorders among partial term pregnancies as a result of this assumption.
+
+  - Note that an alternative strategy would be to assume the same first trimester ANC attendance rate for partial term pregnancies as full term pregnancies 
+    (equal to ANC4 proportion), but this would require us to use different ANC attendance rates for each pregnancy term duration in the already documented and 
+    implemented ANC attendance module. Also note that the data on ANC attendance in DHS is collected on live and still births (full term pregnancies) only.
```

Comment:
I'm not sure I understand this comment. Specifically, why "(equal to ANC4 proportion)"? And what exactly does "use different ANC attendance rates for each pregnancy term duration in the already documented and implemented ANC attendance module" mean?